src/vmware/azext_vmware/_help.py

```diff
@@ -151,3 +156,23 @@
     type: command
     short-summary: Return trial status for subscription by region.
 """
+
+helps['vmware datastore create'] = """
+    type: command
+    short-summary: Create a datastore in a private cloud cluster.

```

Comment:
pls add examples to help

Reply:
@VamsiSamgam, can you add the requested examples?

Reply:
@yungezz  examples have been added , Can you please take a look

Reply:
Examples added.